packages/amplify-codegen/tests/commands/__snapshots__/models.test.js.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`command-models-generates models in expected output path android: Should generate models from a single schema file 1`] = `Array []`;
```

Comment:
Will be fixed when `generateModels` is changed to write to file system instead of return string map.